addons/website/static/src/scss/website.scss

```diff
@@ -199,6 +199,11 @@ font[class*='bg-'] {
         border-top-left-radius: $size;
         border-bottom-right-radius: $size;
     }
+    &.rounded-circle-empty {
+        @extend .rounded-circle;
+        border: 2px solid #dddddd;
```

Comment:
Probably need bootstrap variables instead

Reply:
For the color? Done

Reply:
```scss
border: (2 * $border-width) solid $gray-400;
```
:+1: